celery/app/utils.py

```diff
@@ -383,7 +383,7 @@ def find_app(app, symbol_by_name=symbol_by_name, imp=import_from_cwd):
             try:
                 found = sym.celery
                 if isinstance(found, ModuleType):
-                    raise AttributeError()
+                    raise AttributeError("attribute 'celery' is module not the instance of celery")
```

Comment:
is the celery module...

Reply:
Thanks! I want to double check,  do you mean `attribute 'celery' is celery module not the instance of celery`?

Reply:
yes